docs/conf.rst

```diff
@@ -0,0 +1,253 @@
+.. _sender_conf:
+
+=============
+Configuration
+=============
+
+When constructing a :ref:`sender <sender>` you can pass a configuration string
+to the :func:`questdb.ingress.Sender.from_conf` method.
+
+.. code-block:: python
+
+    from questdb.ingress import Sender
+
+    conf = "http::addr=localhost:9009;username=admin;password=quest;"
+    with Sender.from_conf(conf) as sender:
+        ...
+
+The format of the configuration string is::
+
+    <protocol>::<key>=<value>;<key>=<value>;...;
+
+.. note::
+
+    * The keys are case-sensitive.
+    * The trailing semicolon is mandatory.
+
+The valid protocols are:
+
+* ``tcp``: ILP/TCP
+* ``tcps``: ILP/TCP with TLS
+* ``http``: ILP/HTTP
+* ``https``: ILP/HTTP with TLS
+
+If you're unsure which protocol to use, see :ref:`sender_which_protocol`.
+
+Only the ``addr=host:port`` key is mandatory. It specifies the hostname and port
+of the QuestDB server.
+
+Connection
+==========
+
+* ``addr`` - ``str``: The address of the server in the form of
+  ``host:port``.
+
+  This key-value pair is mandatory, but the port can be defaulted.
```

Comment:
Indention - intentional?

Reply:
Yes. It appears as part of the bullet point.